tests/unit/lib/Dispatcher.js

```diff
@@ -20,7 +20,7 @@ describe('Dispatchr', function () {
     });
 
     it('should not bleed between requires', function () {
-        var Dispatcher2 = require('../../../lib/Dispatcher')();
+        var Dispatcher2 = require('../../../index').create();
         expect(Dispatcher2.isRegistered(mockStore)).to.equal(false);
```

Comment:
This test passes because the registered store don't leek, there is no classes only objects inheriting from on an other:

```js
// This is the root Dispatchr object
 var Dispatchr = require('dispatchr');

// An instance of the dispatcher without context
var dispatcher1 = Dispatchr.create();

// Registering a store
dispatcher1.registerStore(mockStore);
dispatcher1.isRegistered(mockStore) === true;

// Create a dispatcher1 instance with a context
var myDispatcher = dispatcher1.create(context);
myDispatcher.isRegistered(mockStore) === true;

// Creates a Dispatchr instance with a context
var dispatcher2 = Dispatchr.create(context); 

// the store from dispatcher1 is not registered 
dispatchr2.isRegistered(mockStore) === false;

// but you can register any store you like:
dispatcher2.registerStore(mockStore);
dispatcher2.registerStore(delayedStore);
```